fix(skeleton): Change skeleton animation to fade in and out (#1084)
Workday/canvas-kit: modules/docs/mdx/5.0-MIGRATION-GUIDE.mdx

  )
}
```

### Skeleton

> Skeleton was already implemented as a compound component, but there are two notable changes: its
> imports and the animation.

Reviewer: ```suggestion
Skeleton was already implemented as a compound component in v4, but we've made changes to its
imports and to its animation in v5.
```

First, the imports for its sub-components have been updated to be keys on `Skeleton`, for Header,
Shape, and Text. You will now always import just `Skeleton` component, and you may still compose your

> own Skeleton using whatever parts you need.

Reviewer: ```suggestion
The imports for its subcomponents in v4 (`SkeletonHeader`, `SkeletonText`, and `SkeletonShape`) have
been converted to keys on `Skeleton` in v5 (`Skeleton.Header`, `Skeleton.Text`, and
`Skeleton.Shape`). You only need to import the `Skeleton` component in v5, and you may still compose
your own Skeleton using whatever parts you need.
```

> ```tsx
> // v4
> import {Skeleton, SkeletonHeader, SkeletonShape, SkeletonText} from '@workday/canvas-kit-react/skeleton';
> 
> const MySkeleton = () => (
>   <Skeleton>
>     <SkeletonHeader />
>   </Skeleton>
> )
> 
> // v5
> import {Skeleton} from '@workday/canvas-kit-react/text-input';
> 
> const MySkeleton = () => (
>   <Skeleton>
>     <Skeleton.Header />
>   </Skeleton>
> )
> ```

Reviewer: ```suggestion
```tsx
// v4
import {
  Skeleton,
  SkeletonHeader,
  SkeletonShape,
  SkeletonText,
} from '@workday/canvas-kit-react/skeleton';
const MySkeleton = () => (
  <Skeleton>
    <SkeletonHeader />
    <SkeletonText />
    <SkeletonShape width={40} height={40} />
  </Skeleton>
);

// v5
import {Skeleton} from '@workday/canvas-kit-react/skeleton';
const MySkeleton = () => (
  <Skeleton>
    <Skeleton.Header />
    <Skeleton.Text />
    <Skeleton.Shape width={40} height={40} />
  </Skeleton>
);
```
```

> The skeleton animation has been updated from a diagonal sheen, or shimmer, to fading the opacity of
> the entire shape(s) in and out.

Reviewer: ```suggestion
Additionally, the skeleton animation has been updated from a diagonal sheen, or shimmer, to fading
the opacity of the entire shape(s) in and out.
```
